static/js/hotspots.js

```diff
@@ -77,6 +77,9 @@ export function post_hotspot_as_read(hotspot_name) {
             blueslip.error(err.responseText);
         },
     });
+
+    // Remove the read hotspot from the array.
+    page_params.hotspots = page_params.hotspots.filter((hotspot) => hotspot.name !== hotspot_name);
```

Comment:
I would expect logic of this form to below in the `server_events_dispatch.js` logic, so that it updates other browser tabs.

Reply:
I remove the above logic and did [this](https://github.com/zulip/zulip/pull/17557/commits/d14d2bf2a79f05055c72d12a5ac71ebf8336b23f#diff-bd159b0bb92260f1b56582eed341b79bd5f100596458e254b992d067385d8de6R284) to remove non-intro hotspots if in another tab they are marked as read.

Here is the video demo,

https://user-images.githubusercontent.com/63820270/117638491-8178e680-b1a0-11eb-9d7a-92bf7dda6421.mov